tests/testthat/_snaps/layer.md

```diff
@@ -25,7 +25,13 @@
 
 ---
 
-    `x` must be either a string or a <geom> object, not an environment.
+    `environment()` must be either a string or a <geom> object, not an environment.
+
```

Comment:
This seems like a regression in the error message?

Reply:
I'd argue the message has become more informative. The message comes from this test:

```r
expect_snapshot_error(validate_subclass(environment(), "geom"))
```

And we now use `caller_arg(x)` for messaging.

Reply:
Oh - Ignore my pre-coffee eyes